django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import unicode_literals
+
+import gzip
+import os
+import re
+from difflib import SequenceMatcher
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ValidationError
+from django.utils import lru_cache
+from django.utils.encoding import force_text
+from django.utils.html import format_html
+from django.utils.module_loading import import_string
+from django.utils.six import string_types
+from django.utils.translation import ugettext as _
+
+
+@lru_cache.lru_cache(maxsize=None)
+def get_default_password_validators():
+    return get_password_validators()
+
+
+def get_password_validators(validator_config=None):
+    validators = []
+    if validator_config is None:
```

Comment:
If you're going to provide `get_default_password_validators`, why not move this into that function and make `get_password_validators` entirely settings-independent, with the config arg required?